schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column not null and autoincremented.
```
id int NOT NULL IDENTITY(1,1)
```
- [ ] [OPTIONAL] Kindly mark the id column as a primary key.
```
PRIMARY KEY (id)
```